### PR TITLE
SC-9192 BC-69 BC-83 add versioned basepath

### PR DESCRIPTION
**Pull request by tofixx**
# Description
This PR is about updating the server callback route. 

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-83

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by virgilchiriac — APPROVED** [review]

File: ansible/roles/schulcloud-calendar/templates/configmap.yml.j2 [hunk 1]
```diff
@@ -9,7 +9,7 @@ data:
   NODE_ENV: "production"
   CORS_ORIGIN: "http://calendar-svc:3000"
   # Paths
-  SCHULCLOUD_BASE_PATH: "http://api-svc:3030"
+  SCHULCLOUD_BASE_PATH: "http://api-svc:3030/api/v1"
   NOTIFICATION_SERVICE_BASE_PATH: "http://notification-svc"
   SCHULCLOUD_ALL_USERS_FOR_UUID: "http://api-svc:3030/resolve/users/"
   SCHULCLOUD_ALL_SCOPES_FOR_TOKEN: "http://api-svc:3030/resolve/scopes/"
```
Comment (mamutmk5) [inline ansible/roles/schulcloud-calendar/templates/configmap.yml.j2:15]: What is her with api?
```suggestion
  SCHULCLOUD_ALL_USERS_FOR_UUID: "http://api-svc:3030/api/v1/resolve/users/"
  SCHULCLOUD_ALL_SCOPES_FOR_TOKEN: "http://api-svc:3030/api/v1/resolve/scopes/"
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "schulcloud-calendar",
-  "version": "25.6.1",
+  "version": "25.9.1",
```
Comment (mamutmk5) [inline package.json:3]: If new version then it's 26.9.0 or 25.6.2 and not 25.9.1

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "schulcloud-calendar",
-  "version": "25.6.1",
+  "version": "25.9.1",
```
Comment (mamutmk5) [inline package-lock.json:3]: If new version then it's 26.9.0 or 25.6.2 and not 25.9.1

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ## Unreleased
 
+## [26.9.1] - 2021-08-18
```
Comment (mamutmk5) [inline CHANGELOG.md:12]: If new version then it's 26.9.0 or 25.6.2

**Review by mamutmk5 — CHANGES_REQUESTED** [review]

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**